portfolio/src/main/webapp/index.html

```diff
@@ -158,7 +146,7 @@ <h5 class='header col s12 light outline-text' id='bolded'>Awards</h5>
     <div class='parallax'><img src='images/background10.JPG' alt='Unsplashed background img 4'></div>
   </div>
 
-  <!-- 8th (white) container -->
+  <!-- 7th (white) container -->
```

Comment:
The comments with "6th, 7th, 8th, ..." will require updates whenever you add a new container. So it is better to add the comment corresponding to the context stored in the container (e.g. Image container of a background image of flower). Similarly, the file name "background10.JPG" is hard to link to image used on the website. Good naming would be much more appreciated by the person who maintains the website in the future.